src/components/SpaceDelegateRegistryV2Row.vue

```diff
@@ -0,0 +1,123 @@
+<script setup lang="ts">
+import { ref } from 'vue';
+import { validateForm } from '@/helpers/validation';
+import { clone } from '@snapshot-labs/snapshot.js/src/utils';
+
+type DelegateRowForm = {
+  to: string;
+  weight: number;
+};
+
+const props = defineProps<{
+  address: string;
+  weight: number;
+}>();
+
+const emit = defineEmits(['deleteDelegate', 'update:modelValue']);
```

Comment:
Don't understand why you are doing emit `update:modelValue` and not using modelValue prop or v-model

Reply:
Thank you for your feedback. We chose to emit update:modelValue to internally handle value changes before updating the parent component, especially to process and round the weight properly.

I understand the importance of following project conventions and am willing to adjust my code to better align with standard practices, using v-model or modelValue directly.

Are there any specific guidelines or examples within the project I should follow? I’m open to suggestions and happy to make the necessary changes.